scene/gui/rich_text_label.cpp

```diff
@@ -1881,7 +1910,10 @@ void RichTextLabel::_bind_methods() {
 	ADD_GROUP("BBCode", "bbcode_");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "bbcode_enabled"), "set_use_bbcode", "is_using_bbcode");
 	ADD_PROPERTY(PropertyInfo(Variant::STRING, "bbcode_text", PROPERTY_HINT_MULTILINE_TEXT), "set_bbcode", "get_bbcode");
+
+
```

Comment:
Whoops! I ran the file through clang-format in Atom, but I most have forgot to run it again after I made changes.

I set up my pre-commit hook, so hopefully I won't have this issue in the future :sweat_smile: